src/components/Header.js

```diff
@@ -32,19 +32,7 @@ export default function(store){
       toggleView() {
         store.commit('viewToggled');
       },
-      toggleFilterEvents() {
```

Comment:
There is some corresponding code in the template that can also be chopped, the attribute reading `filterEventsTop` (also that definition in this file https://github.com/aclu-people-power/map/pull/69/commits/f96573df19d020dd5aa6e0348d2d1859bc90df8c#diff-6f7a94b64ddda80f88c2971ba5b8b1a3L17)